src/models/account.cpp

```diff
@@ -633,7 +633,7 @@ int Account::importFromQIF(const std::string& path)
     int imported{ 0 };
     unsigned int nextId{ getNextAvailableTransactionId() };
     std::ifstream file{ path };
-    Transaction* transaction{ new Transaction(nextId) };
+    std::shared_ptr<Transaction> transaction = std::make_shared<Transaction>(nextId);
```

Comment:
Sorry. Use `{}-initalization`

```cpp
std::shared_ptr<Transaction> transaction{ std::make_shared<Transaction>(nextId) };
```

Reply:
I think I should leave c++ and never come back to that again 😅

Reply:
Haha sorry. It's just so I can keep a consistent style across the code base :)